UnitySDK/Assets/ML-Agents/Scripts/Agent.cs

```diff
@@ -592,7 +571,7 @@ void UpdateSensors()
         /// <param name="buffer"> A float array that will be used as buffer when generating the observations. Must
         /// be at least the same length as the total number of uncompressed floats in the observations</param>
         /// <param name="adapter"> The WriteAdapter that will be used to write the ISensor data to the observations</param>
-        /// <param name="observations"> A list of observations outputs. This argument will be modified by this method.</param>//  
+        /// <param name="observations"> A list of observations outputs. This argument will be modified by this method.</param>//
         public static void GenerateSensorData(List<ISensor> sensors, float[] buffer, WriteAdapter adapter, List<Observation> observations)
```

Comment:
Not done here because the other PR needs it, but this gets rid of the calls to GenerateSensorData